src/translation/Listing.js

```diff
@@ -63,6 +81,96 @@ Vue.component('translation-listing', {
         editTranslation (item) {
             this.$modal.show('edit-translation', item);
         },
+
+        showImport () {
+            this.$modal.show('import-translation');
+        },
+
+        showExport () {
+            this.$modal.show('export-translation');
+        },
+
+        nextStep(){
+            if(this.currentstep === 1){
+                return this.$validator.validateAll()
+                    .then(result => {
+                        if (!result) {
+                            this.$notify({ type: 'error', title: 'Error!', text: 'The form contains invalid fields.'});
+                            return false;
+                        }
+
+                        let url = '/admin/translations/import';
```

Comment:
Do we need to hardcode the url here? I do not like this approach, should be provided as input.